packages/repository/src/model.ts

```diff
@@ -30,12 +31,20 @@ export interface PropertyDefinition {
   type: PropertyType; // For example, 'string', String, or {}
   id?: boolean | number;
   json?: PropertyForm;
-  jsonSchema?: {[attribute: string]: any};
+  jsonSchema?: PropertyJsonSchema;
```

Comment:
I believe we are expecting vanilla JSON Schema here, not the OpenAPI flavor. IIRC, there are subtle differences between the two and we have a helper to convert from JSON Schema to OpenAPI Schema:

https://github.com/strongloop/loopback-next/blob/267b074a93dc7483333486e2b381b3d7168ebc79/packages/openapi-v3/src/json-to-schema.ts#L37-L43

Please use type definitions from `json-schema` and/or `@types/json-schema`, do not use `openapi3-ts`.

Reply:
Nice to have (feel free to ignore):

While you are changing this line, can you please add a tsdoc comment to explain what is `jsonSchema` for?

Reply:
AFAIK, buthe `jsonSchema` is mainly being used to populate the final `SchemaObject` in `openapi.json`. If that's the case, at the bottom off [this section](https://github.com/OAI/OpenAPI-Specification/blob/9218ce60424b03794411a8c6d0454c1c6d31aba0/versions/3.0.2.md#properties), it states:

```
Additional properties defined by the JSON Schema specification that are not mentioned here are strictly unsupported.
```